plat/unix/update_tags.sh

```diff
@@ -69,9 +69,24 @@ echo $$ > "$TAGS_FILE.lock"
 # Remove lock and temp file if script is stopped unexpectedly.
 trap 'errorcode=$?; rm -f "$TAGS_FILE.lock" "$TAGS_FILE.temp"; exit $errorcode' INT QUIT TERM EXIT
 
+# Change to directory of tags file for relative paths.
+cd "$(dirname "$TAGS_FILE")"
+
 INDEX_WHOLE_PROJECT=1
 if [ -f "$TAGS_FILE" ]; then
     if [ "$UPDATED_SOURCE" != "" ]; then
+        # Make the file path relative to the tags file.
+        # This is required for consistent naming.
+        # See https://github.com/ludovicchabant/vim-gutentags/issues/70.
+        relpath() {
+            python -c "import os.path; print(os.path.relpath('$1', '${2}'))"
```

Comment:
This is not going to work if path contains backslash or single quote. Should be

    A="$1" B="$2" python -c 'import os; print(os.path.relpath(os.environ["A"], os.environ["B"]))'

(you may also pass this as an argument after `-c` command, but this would require two imports).